lib/12T/xschem/extspice.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# extspice.py
```

Comment:
can you make sure this confirm to https://peps.python.org/pep-0008/?

Reply:
not needed - deleted.